src/stream/src/executor/backfill/cdc/cdc_backfill.rs

```diff
@@ -201,9 +207,26 @@ impl<S: StateStore> CdcBackfillExecutor<S> {
             .await
             .expect("Retry create cdc table reader until success.")
         });
-
+        let timestamp_handling: Option<TimestampHandling> = self
+            .properties
+            .get("debezium.time.precision.mode")
+            .map(|v| v == "connect")
+            .unwrap_or(false)
+            .then_some(TimestampHandling::Milli);
+        let timestamptz_handling: Option<TimestamptzHandling> = self
+            .properties
+            .get("debezium.time.precision.mode")
+            .map(|v| v == "connect")
+            .unwrap_or(false)
+            .then_some(TimestamptzHandling::Milli);
```

Comment:
This reminds me of another issue. Before any fix, could you add a `time` column to the tests?

The test is expected to fail, because `debezium.time.precision.mode = connect` also sets `TimeHandling::Milli` in the upstream, while our parser is currently hardcoded to `TimeHandling::Micro` (assuming `debezium.time.precision.mode = adaptive_time_microseconds`)

Reply:
Good catch!